app/assets/javascripts/table_sorting.coffee

```diff
@@ -2,18 +2,19 @@
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
 
-jQuery ->
-  $(document).on 'ready page:load', ->
-    # make the columns of all of the tables sortable
-    autosortable_tables = [ '#call_list table', '#completed_calls table', '#urgent_patients table', '#user-list table']
-    autosortable_tables.forEach ( table ) ->
-        stupidtable = $( table ).stupidtable()
+$(document).on 'turbolinks:load', ->
```

Comment:
This fixes a bug when you navigate between pages and the stupidtable no longer works.